KB created: How to Whitelist the Elastic AWS IPs for Telerik Platform

Mike4o opened the pull request.

Commit:
KB created: How to Whitelist the Elastic AWS IPs for Telerik Platform

ikoevska commented on how-to-whitelist-the-elastic-aws-ips-for-telerik-platform.md:
```diff
@@ -0,0 +1,47 @@
+title: How to Whitelist the Elastic AWS IPs for Telerik Platform
+description: Information for establishing a secure connection between your machine and Amazon Web Services AppBuilder stays behind.
+type: how-to
+page_title: Whitelisting Amazon IPs for secure AppBuilder connection
```
Secure ... Connection

ikoevska commented on how-to-whitelist-the-elastic-aws-ips-for-telerik-platform.md:
```diff
@@ -0,0 +1,47 @@
+---
+title: How to Whitelist the Elastic AWS IPs for Telerik Platform
+description: Information for establishing a secure connection between your machine and Amazon Web Services AppBuilder stays behind.
```
You can drop the "information for" part and go with "Establishing a secure connection between your machine and the Amazon Web Services that AppBuilder stays behind."

ikoevska commented on how-to-whitelist-the-elastic-aws-ips-for-telerik-platform.md:
```diff
@@ -0,0 +1,47 @@
+</table>
+
+## Description
+Is there a way to whitelist the range of IP addresses that Telerik Platform uses on the Amazon Web Services infrastructure? How to do this having in mind that AWS IPs are elastic? I need this in order to establish a secure communication channel.
```
I would probably go about rewriting this into something like:

"The Telerik Platform is hosted on the Amazon Web Services and uses a range of elastic IP addresses. If you are behind a firewall, you might need to whitelist this range of IPs to establish a secure connection between your system and the Telerik Platform." or something like that.

ikoevska commented on how-to-whitelist-the-elastic-aws-ips-for-telerik-platform.md:
```diff
@@ -0,0 +1,47 @@
+Is there a way to whitelist the range of IP addresses that Telerik Platform uses on the Amazon Web Services infrastructure? How to do this having in mind that AWS IPs are elastic? I need this in order to establish a secure communication channel.
+
+## Solution
+> **IMPORTANT**: Please do have in mind that Amazon Web Services IP ranges are subject to change without notice. This is  further discussed [here](http://serverfault.com/questions/665191/whitelist-elastic-load-balancer-based-on-ip).
```
You can safely drop the "Please do have in mind that"

Generally, using "please" should be avoided. There are a few cases where you might need to use it - typically in cases where you're causing some inconvenience that you are aware of. However, "please" should appear sparingly, if at all.

ikoevska commented on how-to-whitelist-the-elastic-aws-ips-for-telerik-platform.md:
```diff
@@ -0,0 +1,47 @@
+## Solution
+> **IMPORTANT**: Please do have in mind that Amazon Web Services IP ranges are subject to change without notice. This is  further discussed [here](http://serverfault.com/questions/665191/whitelist-elastic-load-balancer-based-on-ip).
+
+The following information should help you or your system administrator whitelest the elastic load-balancer based on the AWS IP range that Telerik Platform stays behind:
```
In this case, it's good to go with "help to". Otherwise, this becomes hard to read.

"This information will help you or your system administrator to whitelist the elastic load balancer based on the AWS IP range..."

Btw, it's a good idea on your first mention of Amazon Web Services, to note the abbreviation in brackets.

ikoevska commented on how-to-whitelist-the-elastic-aws-ips-for-telerik-platform.md:
```diff
@@ -0,0 +1,47 @@
+
+The following information should help you or your system administrator whitelest the elastic load-balancer based on the AWS IP range that Telerik Platform stays behind:
+
+* The name of the load-balancer Telerik Platform services use is **_nginx7layerroutinglb-223465912.us-east-1.elb.amazonaws.com_**.
```
load balancer that the Telerik Platform services use

Also, in this entire article, you can use variables for the product names instead of hardcoding them.

ikoevska commented on how-to-whitelist-the-elastic-aws-ips-for-telerik-platform.md:
```diff
@@ -0,0 +1,47 @@
+The following information should help you or your system administrator whitelest the elastic load-balancer based on the AWS IP range that Telerik Platform stays behind:
+
+* The name of the load-balancer Telerik Platform services use is **_nginx7layerroutinglb-223465912.us-east-1.elb.amazonaws.com_**.
+* The load-balancer is situated in the **us-east-1** region.
```
load balancer

ikoevska commented on how-to-whitelist-the-elastic-aws-ips-for-telerik-platform.md:
```diff
@@ -0,0 +1,47 @@
+* The name of the load-balancer Telerik Platform services use is **_nginx7layerroutinglb-223465912.us-east-1.elb.amazonaws.com_**.
+* The load-balancer is situated in the **us-east-1** region.
+* The complete list of AWS IP addresses can be found [here](https://ip-ranges.amazonaws.com/ip-ranges.json). 
+* Whitelist all the IPs from the above list, corresponding to the **us-east-1** region.
```
All other bullet points are somewhat uniform (they use identical phrasing and have identical structure). Consider reworking this bullet point to be similar to the others. Alternatively, consider moving this point to a separate paragraph.

ikoevska commented on how-to-whitelist-the-elastic-aws-ips-for-telerik-platform.md:
```diff
@@ -0,0 +1,47 @@
+* The complete list of AWS IP addresses can be found [here](https://ip-ranges.amazonaws.com/ip-ranges.json). 
+* Whitelist all the IPs from the above list, corresponding to the **us-east-1** region.
+
+Domains used by Telerik Platform services:
```
From this line below, I am a bit confused as to what is happening. Is this to be whitelisted in some way? Is this information a part of the bulleted list above? IMHO, this needs to be reworked a bit.

Commit:
Addressing comments

ikoevska commented on how-to-whitelist-the-elastic-aws-ips-for-telerik-platform.md:
```diff
@@ -0,0 +1,50 @@
+</table>
+
+## Description
+Is there a way to whitelist the range of IP addresses that Telerik Platform uses on the {{ page.aws }}  (AWS) infrastructure? How to do this having in mind that AWS IPs are elastic? I need this in order to establish a secure communication channel.
```
Everything looks good to me but this paragraph. I am bothered by the list of questions and the "I" sentence. Perhaps Evgeni Boevski should take a look as well. 

Keep in mind that if this is the style we're after for the description, all other KBs should use similar phrasing.

Commit:
Addressing comments

- Moved the questions to the description meta filed and added proper
text for the Description area
